src/python/CMakeLists.txt

```diff
@@ -16,8 +16,11 @@ if (SWIG_FOUND)
 
   set(swig_files
     Angle
+    Color
     GaussMarkovProcess
     Rand
+    RollingMean
+    Spline
```

Comment:
I just noticed that this `swig_files` variable isn't actually used and I think we can delete it?

Reply:
you are right, I will delete it.